lib/components/Editor/CustomExtensions/Image/ImageComponent.jsx

```diff
@@ -1,35 +1,107 @@
-import React from "react";
+import React, { useRef } from "react";
 
-import { NodeViewWrapper } from "@tiptap/react";
+import { NodeViewContent, NodeViewWrapper } from "@tiptap/react";
+import classNames from "classnames";
+import { isEmpty } from "lodash";
+import { MenuHorizontal } from "neetoicons";
 import { Resizable } from "re-resizable";
 
-const ImageComponent = ({ node, editor, getPos }) => {
-  const { alt, src, float, align, height, width } = node.attrs;
-  const caption = alt || "";
+import Dropdown from "components/Common/Dropdown";
+import MenuButton from "components/Common/MenuButton";
+import { humanize } from "utils/common";
+
+import { buildImageOptions } from "./utils";
+
+const { Menu } = Dropdown;
+
+const ImageMenu = ({ align, updateAttributes, deleteNode }) => {
+  const menuOptions = buildImageOptions();
+
+  const handleClick = align => {
+    align ? updateAttributes({ align }) : deleteNode();
+  };
+
+  return (
+    <Dropdown
```

Comment:
Instead of having a primary button for the dropdown, please use the secondary button.